Symfony/CS/Tests/Console/ConfigurationResolverTest.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\CS\Tests\Console;
 
+use Guzzle\Common\Exception\UnexpectedValueException;
```

Comment:
It is used to resolve this class:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1241/files#diff-84cef3eba62d0b0d72bf5973c56ef3deR463

(this prevent false positives as the code being tested throws an exception with the class name `UnexpectedValueException` but in another namespace)